docs/embedding/sdk/next-js.md

```diff
@@ -8,7 +8,7 @@ title: Embedded analytics SDK - Using the SDK with Next.js
 
 {% include plans-blockquote.html feature="Embedded analytics SDK" sdk=true %}
 
-Some notes on using the Embedded analytics SDK with [Next.js](https://nextjs.org/).
+Some notes on using the Embedded analytics SDK with [Next.js](https://nextjs.org/). The SDK is tested to work with Next.js 14, although it may work with earlier versions. Next.js 15 is cuirrently not supported.
```

Comment:
```suggestion
Some notes on using the Embedded analytics SDK with [Next.js](https://nextjs.org/). The SDK is tested to work with Next.js 14, although it may work with earlier versions. Next.js 15 is currently not supported.
```